modules/data.remote/DESCRIPTION

```diff
@@ -1,12 +1,15 @@
 Package: PEcAn.data.remote
 Type: Package
-Title: PEcAn functions used for remote sensing
+Title: PEcAn functions used for extracting remote sensing data
 Version: 1.7.0
-Date: 2018-12-03
-Authors@R: c(person("Mike","Dietze"))
-Author: Mike Dietze
-Maintainer: Mike Dietze <dietze@bu.edu>
-Description: PEcAn module for processing remote data
+Date: 2019-02-06
+Authors@R: c(person("Mike","Dietze"), person("Bailey", "Morrison"))
+Author: Mike Dietze, Bailey Morrison
+Maintainer: Bailey Morrison <bmorrison@bnl.gov>
+Description: PEcAn module for processing remote data. Python module requirements: requests, json, re, ast, panads, sys. If any of these modules are missing, install using pip install <module name>.
+Depends:
+    MODISTools
+    reticulate
```

Comment:
Can these be moved to Imports? We're trying to push PEcAn toward using Imports rather than Depends everywhere it's possible. So far it's usually been possible, but I don't know `reticulate` well enough to say for sure.

Reply:
@bailsofhay sorry, I keep getting these backward.  As @infotroph says put into imports instead of depends, unless that doesnt work.  I was confused and thought we were going the other way but then I recalled the whole recent thread on build, depends, etc.  My bad!

Reply:
Fixed!